docs/FAQ.rst

```diff
@@ -31,4 +32,37 @@ This will be because you have a version of setuptools that is too old, Gilbert r
 
 You can fix this by updating versions:
 
-    pip install -U setuptools
\ No newline at end of file
+.. code-block:: sh
+
+    pip install -U setuptools
+	
+How do I have Gilbert automatically rebuild the site when files change?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Have gilbert watch your files, and re-render on changes:
```

Comment:
Can we rearrange this to say "If you have Linux, you can use the build in watch command" sort of approach, and propose watchexec for every other OS [MacOS, Windows, even Linux too :) ]

Reply:
Sure, thanks for the feedback! I originally thought that it was windows specific, so I just updated it to be more general for every other OS